src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -137,6 +138,8 @@ Rectangle {
             icon.height: 21
             type: StatusFlatRoundButton.Type.Secondary
 
+            property alias tooltip: tooltip
```

Comment:
This is enough, no need to expose it explicitly on the chat toolbar

Please do the same for all other action buttons as well.

Reply:
This wont work because of the way in which QMl does binding. It does not let you access the property newly created in this component but links it the actual StatusToolTip.qml component. The only way this can work is if it in a file and that file is then aliased!
You could try it out very simply. wont work!

Reply:
Ah, then what if we do `property var tooltip` instead and assign the tooltip in the buttons `onCompleted` handler (for now). Once we've landed tooltips in StatusBaseButton we can refactor this (essentially just removing code).

Reply:
As discussed offline, @Khushboo-dev-cpp pointed out that this solution will result with the same issue as the other approach. We'll roll back and put tooltip instances into flat round button so they are a natural property of the buttons which are then automatically exposed via chat tool bar.

Reply:
Done!